src/bci_build/package/keadhcp.py

```diff
@@ -0,0 +1,38 @@
+"""KEA DHCP Server BCI container"""
+
+from bci_build.os_version import ALL_NONBASE_OS_VERSIONS
+from bci_build.os_version import CAN_BE_LATEST_OS_VERSION
+from bci_build.package import ApplicationStackContainer
+from bci_build.package import ParseVersion
+from bci_build.package.versions import format_version
+from bci_build.package.versions import get_pkg_version
+
+_BASE_PODMAN_KEA_CMD = "podman run --replace -it --privileged --network=host"
+_KEA_DHCP4_CONFIG_PATH = "/etc/kea/kea-dhcp4.conf"
+_KEA_DHCP6_CONFIG_PATH = "/etc/kea/kea-dhcp6.conf"
+
+KEA_DHCP_CONTAINERS = []
+
+for os_version in ALL_NONBASE_OS_VERSIONS:
+    KEA_DHCP_CONTAINERS.append(
+        ApplicationStackContainer(
+            name="keadhcp",
+            os_version=os_version,
+            version=format_version(
+                kea_ver := get_pkg_version("keadhcp", os_version), ParseVersion.MAJOR
+            ),
+            tag_version=format_version(kea_ver, ParseVersion.MINOR),
+            is_latest=os_version in CAN_BE_LATEST_OS_VERSION,
```

Comment:
please add the `license="MPL-2.0"` somewhere here.

Reply:
added